man/udevadm.xml

```diff
@@ -308,8 +308,9 @@
           <term><option>-y</option></term>
           <term><option>--sysname-match=<replaceable>PATH</replaceable></option></term>
           <listitem>
-            <para>Trigger events for devices with a matching sys
-            device path. This option can be specified multiple times
+	    <para>Trigger events for devices with a matching
+	    the last component (the filename) of a sys device path.
```

Comment:
this isn't proper english: "with a matching the last component" isn't right. should probably be "with the last component (i.e. the file name) of a <filename>/sys</filename> device path matching" or so